diff -Naur libuae/zfile_archive.cpp libuae/zfile_archive.cpp
--- libuae/zfile_archive.cpp	2012-02-13 22:09:01.000000000 +0100
+++ libuae/zfile_archive.cpp	2012-02-15 18:07:06.512866167 +0100
@@ -16,7 +16,7 @@
 
 #include "options.h"
 #include "zfile.h"
-#include "archivers/zip/unzip.h"
+#include <minizip/unzip.h>
 #include "archivers/dms/pfile.h"
 #include "crc32.h"
 #include "zarchive.h"
diff -Naur libuae/zfile.cpp libuae/zfile.cpp
--- libuae/zfile.cpp	2012-02-13 22:08:58.000000000 +0100
+++ libuae/zfile.cpp	2012-02-15 18:06:57.392866167 +0100
@@ -25,7 +25,7 @@
 #include "diskutil.h"
 #include "fdi2raw.h"
 
-#include "archivers/zip/unzip.h"
+#include <minizip/unzip.h>
 #include "archivers/dms/pfile.h"
 #include "archivers/wrp/warp.h"
 
diff -Naur Makefile Makefile
--- Makefile	2012-02-13 21:39:43.000000000 +0100
+++ Makefile	2012-02-15 18:08:31.052866167 +0100
@@ -42,7 +42,7 @@
 		-I$(libfsemu_dir)/include
 ldflags =
 libs = -L$(libfsemu_dir)/out -lfsemu `sdl-config --libs` \
-		`pkg-config --libs glib-2.0 gthread-2.0` -lpng -lz
+		`pkg-config --libs glib-2.0 gthread-2.0` -lpng -lz -lminizip
 
 libuae_warnings = -Wno-unused-value -Wno-uninitialized -Wno-sign-compare
 generate = 0
